alloc.c

```diff
@@ -406,15 +406,15 @@ bool output_choose_layers(struct liftoff_output *output,
 		}
 
 		/* Try to use this layer for the current plane */
-		liftoff_log(LIFTOFF_DEBUG, "Layer %p -> plane %"PRIu32": "
+		liftoff_log(LIFTOFF_DEBUG, "  Layer %p -> plane %"PRIu32": "
 			    "applying properties...",
 			    (void *)layer, plane->id);
 		if (!plane_apply(plane, layer, result->req, &compatible)) {
 			return false;
 		}
 		if (!compatible) {
 			liftoff_log(LIFTOFF_DEBUG,
-				    "Layer %p -> plane %"PRIu32": "
+					"  Layer %p -> plane %"PRIu32": "
```

Comment:
Nitpick: alignment is broken here

Reply:
These tabs are killing me sometimes. I am using QtCreator and most of the time it seems to work fine after I configured it to use tabs. But sometimes the alignment breaks for unknown reasons. I can't find this alignment problem anymore in the current revision.